docs/src/content/docs/guides/i18n.mdx

```diff
@@ -120,7 +120,7 @@ If a translation is not yet available for a language, Starlight will show reader
 
 Some of Starlight’s default UI requires text labels to work.
 For example, the table of contents on this page has an “On this page” heading in English.
-We aim to ship these labels in as many languages as possible but currently only have support for English, German, Japanese, and Spanish.
+We aim to ship these labels in as many languages as possible but currently only have support for English, German, Japanese, Spanish, and Portuguese.
```

Comment:
I guess maybe we can drop “only” now that we have 5 languages. What do you think?

```suggestion
We aim to ship these labels in as many languages as possible. Currently, English, German, Japanese, Portuguese, and Spanish are supported out of the box.
```

Reply:
Agree!